dashboard/src/components/AppView/AppView.tsx

```diff
@@ -14,6 +14,7 @@ import AppDetails from "./AppDetails";
 import AppNotes from "./AppNotes";
 import "./AppView.css";
 import ChartInfo from "./ChartInfo";
+import SecretTable from "./SecretsTable";
```

Comment:
I think that sooner than latter (not for this PR) we should extract the table as a High Order Component where you pass the columns, the resources and potentially a custom `item` component.